tests/ext/test_updater.py

```diff
@@ -755,19 +792,75 @@ async def set_webhook(*args, **kwargs):
                 assert self.message_count == 0
 
             # We call the same logic twice to make sure that restarting the updater works as well
-            await updater.start_webhook(
-                drop_pending_updates=drop_pending_updates,
-                ip_address=ip,
-                port=port,
-                url_path="TOKEN",
-            )
+            if unix:
+                await updater.start_webhook(
+                    drop_pending_updates=drop_pending_updates,
+                    secret_token=secret_token,
+                    unix=file_path,
+                    webhook_url="string",
+                )
+            else:
+                await updater.start_webhook(
+                    drop_pending_updates=drop_pending_updates,
+                    ip_address=ip,
+                    port=port,
+                    url_path="TOKEN",
+                    secret_token=secret_token,
+                    webhook_url="string",
+                )
             assert updater.running
             update = make_message_update("Webhook")
-            await send_webhook_message(ip, port, update.to_json(), "TOKEN")
+            await send_webhook_message(
+                ip,
+                port,
+                update.to_json(),
+                "" if unix else "TOKEN",
+                secret_token=secret_token,
+                unix=file_path if unix else None,
+            )
             assert (await updater.update_queue.get()).to_dict() == update.to_dict()
             await updater.stop()
             assert not updater.running
 
+    @pytest.mark.skipif(
+        platform.system() == "Windows",
+        reason="Windows doesn't support unix bind",
+    )
```

Comment:
these tests should pass on all OSes, shouldn't they?

Reply:
right.